hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -2966,6 +2967,40 @@ public String getRpcPort() {
   }
 
   @Override
+
+  public String getOmRatisRoles() {
+    List<ServiceInfo> serviceList = null;
+    StringBuilder sb = new StringBuilder();
+    try {
+      serviceList = getServiceList();
+    } catch (IOException e) {
+      return "";
```

Comment:
Please print a meaningful error LOG message here instead of returning an empty string

Reply:
Sure will do 
Thanks !!